hydra-main/src/main/java/com/addthis/hydra/job/web/resources/JobsResource.java

```diff
@@ -1279,7 +1281,26 @@ private static Response buildServerError(Exception exception) {
         if (message == null) {
             message = exception.toString();
         }
-        return Response.serverError().entity(message).build();
+
+        String stack = "";
+        for (StackTraceElement s : exception.getStackTrace()) {
+            stack += s + "\n";
+        }
+
+        JSONObject body = new JSONObject();
+        String response;
+
+        try {
+            body.put("error", "A java exception was thrown.");
+            body.put("message", message);
+            body.put("stack", stack);
+            response = body.toString();
+        } catch(JSONException ex) {
+            log.error("[buildServerError]", "failed to construct a json response");
+            response = "{\"error\": \"Failed to serialize java exception. Please see spawn logs for more details.\"}";
```

Comment:
Leaving out the actual exception may lead to confusion later.  The message with the exception would be helpful to know for debugging purposes.